sdk/monitor/monitor-opentelemetry-exporter/src/export/statsbeat.ts

```diff
@@ -9,10 +9,9 @@ import { resourceMetricsToEnvelope } from "../utils/metricUtils";
 import { AzureMonitorBaseExporter } from "./base";
 
 /**
- * @internal
  * Azure Monitor Statsbeat Exporter
  */
-export class _AzureMonitorStatsbeatExporter
+export class AzureMonitorStatsbeatExporter
```

Comment:
Let's move this file to statsbeat folder and rename the file to stastbeatExporter.ts